configure.ac

```diff
@@ -364,7 +364,7 @@ m4_pattern_forbid([^_?PKG_[A-Z_]+$],[*** pkg.m4 missing, please install pkg-conf
 have_dbus=no
 AC_ARG_ENABLE(dbus, AS_HELP_STRING([--disable-dbus], [disable usage of dbus-1 in tests]))
 AS_IF([test "x$enable_dbus" != "xno"], [
-        PKG_CHECK_MODULES(DBUS, [dbus-1 >= 1.3.2],
+        PKG_CHECK_MODULES(DBUS, [dbus-1 >= 1.9.18],
```

Comment:
I think it makes sense to revert that PKG_CHECK_MODULES modules change.

It would also make sense to more fully explain the requirements in the README file.

libdbus >= 1.3.2 needed for tests
dbus >= 1.9.18 needed at runtime, OR pass --with-dbuspolicydir=/etc/dbus-1/system.d.